Bellerophon/BellerophonTests/BellerophonTests.swift

```diff
@@ -10,27 +10,127 @@ import XCTest
 @testable import Bellerophon
 
 class BellerophonTests: XCTestCase {
-    
+
+    enum ResponseCases: Int {
+        case KillSwitchOffForceUpdateOff = 0b00
+        case KillSwitchOffForceUpdateOn = 0b01
+        case KillSwitchOnForceUpdateOff = 0b10
+        case KillSwitchOnForceUpdateOn = 0b11
```

Comment:
I thought using binary would make it easier to understand the relationship between a response case and its raw value. Each digit represents whether a field is turned on/off. It would benefit more if there are more fields in the response, where translation between case and its raw value would be simple, in my opinion :)